backend/e4h-services/ingestion-service/app/utils/organization_service_client.py

```
@@ -141,6 +141,67 @@ def _find_organisation_by_code(
+                timeout=60,
+            )
+            response.raise_for_status()
+            org_users = response.json().get("OrgUsers") or []
+            for org_user in org_users:
+                if org_user.get("isDeleted"):
+                    continue
+                user_id = self._string_value(org_user.get("userId"))
+                if user_id:
+                    return user_id
+            return None
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)